lib/types/SSI.types.ts

```diff
@@ -1,20 +0,0 @@
-import { PresentationDefinitionV1, PresentationDefinitionV2 } from '@sphereon/pex-models';
```

Comment:
This has the potential to break external implementations. That is fine, but then we need to write it down somewhere, so we can include it in the release notes for v2.x

Reply:
I'll update release notes for this change